Makefile

```diff
@@ -78,17 +81,25 @@ bdist: npm-packages
 
 install: bdist lint ## Build distribution and install
 	pip install --upgrade dist/elyra-*-py3-none-any.whl
```

Comment:
The original idea here would be to enable handing a wheel to someone and he would be able to install/update elyra on his system, but after our conversation, it does not seem that this is possible unless the npm packages are published. We should have a conversation and work out what should we do and maybe get rid of these.

Reply:
Based on what we discuss in scrum, I have a couple ideas that would remove quite a bit from the make file and streamline this pr